README.md

```diff
@@ -642,31 +577,30 @@ Note that backslashes in double-quoted strings *must* be double-escaped and
 backslashes in single-quoted strings *may* be double-escaped.
 
 ##### `name`
-(**Namevar**: If ommitted, this attribute's value will default to the resource's
```

Comment:
Do we not mention when things are the namevar in the reference section? Curious mostly.